sdk/core/azure-core/azure/core/pipeline/_base.py

```diff
@@ -194,7 +194,8 @@ def run(self, request, **kwargs):
         :return: The PipelineResponse object
         :rtype: ~azure.core.pipeline.PipelineResponse
         """
-        self._prepare_multipart_mixed_request(request)
+        multipart_options = kwargs.pop("multipart_options", None) or {}
```

Comment:
Two things:
- Why not `kwargs.pop("multipart_options", {})` ?
- Do we want this parameter to be hidden?